bindings/nodejs/examples/1-create-account.js

```diff
@@ -8,9 +8,7 @@ const manager = require('./account-manager');
 
 async function run() {
     try {
-        // await manager.setStrongholdPassword(process.env.SH_PASSWORD);
-        // await manager.storeMnemonic();
-
+        await manager.setStrongholdPassword(process.env.SH_PASSWORD);
```

Comment:
Don't you need to update `account-manager.js` for that so Stronghold is used?
Related to that I think `account-manager.js` should be removed altogether, if you look at the Rust examples it's not necessary anymore to provide all the options every time

Reply:
I left the `account-manager.js` there, although I changed it a bit to reduce code duplications throughout the tests